agent.go

```diff
@@ -1103,6 +1105,8 @@ func newContainerCb(pod *pod, data []byte) error {
 
 	pod.setContainer(payload.ID, container)
 
+	container.wgProcesses.Add(1)
```

Comment:
This can be moved to `runContainerProcess()` in order to avoid calling this both from `newContainerCb()` and `execCb()`. There is no risk to increase the waitgroup counter from this go routine as long as we make sure the counter is incremented before the signal is sent through the `started` Go channel.

Reply:
@sboeuf iirc Add should execute before the statement creating the goroutine and not in the goroutine,  hence the two places.

Reply:
Oh... You think it does not work even if we have the handler `container.wgProcesses` from the goroutine ?
I thought it would be okay since the starting of any process (newContainer or exec) waits for the `started` channel for synchronization.